src/control-plane/frontend/Dockerfile

```diff
@@ -1,7 +1,4 @@
-ARG ADAPTER_VERSION=0.8.2
-FROM --platform=linux/aarch64 public.ecr.aws/awsguru/aws-lambda-adapter:$ADAPTER_VERSION-aarch64 AS adapter
-
-FROM public.ecr.aws/docker/library/node:20 AS builder
+FROM public.ecr.aws/docker/library/node:20
```

Comment:
Should we keep it as builder to save the runtime layer size?

Reply:
This Dockerfile use at build stage, not deploy to lambda.